DEPLOYMENT.md

```diff
@@ -0,0 +1,78 @@
+# MAAP API Query Service Deployment
+### Steps
+1. Clone the repo
+2. Install `aws sam cli`:
+    ``` 
+    brew tap aws/tap
+    brew install aws-sam-cli
+    ```
+3. Install the dependencies:
+    ```
+    npm install
+4. Create `ssm` parameters for the database credentials (`host`, `name`, `username`, `password`). Read: [readme](https://github.com/MAAP-Project/maap-api-query-service#ssm).
+5. Set the environment variables
+    ```
+    `STAGE` (default: `dev`)
+    `NODE_ENV` (default: `production`)
+    `SSM_GEDI_DB_HOST` (default: `/dev/gedi-cal-val-db/host`)
+    `SSM_GEDI_DB_NAME` (default: `/dev/gedi-cal-val-db/name`)
+    `SSM_GEDI_DB_USER` (default: `/dev/gedi-cal-val-db/user`)
+    `SSM_GEDI_DB_PASS` (default: `/dev/gedi-cal-val-db/pass`)
+    ```
+6. Run full deployment (builds, packages and deploys)
+    ```
+    npm run full-deploy
+    ```
+
+### Testing the deployment
+1. Copy the `QueryStateMachineArn Value` from the above step. Looks like 
+   ```
+   arn:aws:states:us-east-1:532321095167:stateMachine:maap-api-query-service-dev-RunQuery
```

Comment:
nit: but in general I try not to commit AWS account numbers. It's not really sensitive information but does make it clear that this is a value users should have to specify themselves